src/Tables/TableCell.tsx

```diff
@@ -1,30 +1,49 @@
 import React, {
-  FunctionComponent, ReactElement, ReactNode, useContext,
+  FunctionComponent,
+  ReactElement,
+  ReactNode,
+  useContext,
 } from 'react';
 import styled, { ThemeContext } from 'styled-components';
 import { BaseTheme } from '../Theme';
 
+export enum ALIGN {
```

Comment:
This could probably do with some jsdoc documentation on it

Reply:
Added jsdoc comment through 7434ce7